README.md

```diff
@@ -2,6 +2,10 @@ Julia
 =====
 A parallel realtime Julia set renderer in Rust
 
+Dependencies
+--------
+Requires the SDL2 library. On Homebrew, you can do `brew install sdl2` to install it.
```

Comment:
Can you include a note that Homebrew is on macOS, mention using package managers in general, and have a link to https://wiki.libsdl.org/Installation?